### PR TITLE
pythonPackages.Markups: disable some failing tests

### PR DESCRIPTION
**Pull request by eduardosm**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by eduardosm** [comment]
Result of `nixpkgs-review pr 122038` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built:</summary>
  <ul>
    <li>python38Packages.Markups</li>
    <li>python39Packages.Markups</li>
    <li>retext</li>
  </ul>
</details>

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 122038` at 78e77832 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built successfully:</summary>
  <ul>
    <li>python38Packages.Markups</li>
    <li>python39Packages.Markups</li>
    <li>retext</li>
  </ul>
</details>

**Comment by eduardosm** [comment]
ZHF: #122042

File: pkgs/development/python-modules/Markups/default.nix [hunk 1]
```diff
@@ -16,9 +17,16 @@ buildPythonPackage rec {
     sha256 = "e309d79dde0935576ce1def6752f2127a12e2c2ea2ae8b0c69f99ff8bc12181d";
   };
 
-  checkInputs = [ markdown docutils pygments ];
   propagatedBuildInputs = [ python-markdown-math ];
 
+  checkInputs = [ pytestCheckHook markdown docutils pygments ];
+  disabledTests = [
+    # these tests fail on the current version
```
[the remark this answers is not in the document]
Comment (eduardosm) [reply pkgs/development/python-modules/Markups/default.nix:24]: https://github.com/NixOS/nixpkgs/pull/121017#issuecomment-834485828

[the remark this answers is not in the document]
Comment (mweinelt) [reply pkgs/development/python-modules/Markups/default.nix:24]: @SuperSandro2000 Did you look into these tests?

[the remark this answers is not in the document]
Comment (SuperSandro2000) [reply pkgs/development/python-modules/Markups/default.nix:24]: No, not really.

**Comment by jonringer** [comment]
gah, sorry, forgot about this PR :(

**Pull request closed without merging**